electron/gog/library.ts

```diff
@@ -628,6 +628,9 @@ export class GOGLibrary {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public readInfoFile(appName: string, installPath?: string): any {
     const gameInfo = this.getGameInfo(appName)
+    if (!gameInfo) {
```

Comment:
this is necessary because if there was no cache, heroic was not loading gog games, this forces it to create the cache.